libraries/AP_HAL_ChibiOS/hwdef/GEPRCF745BTHD/defaults.parm

```diff
@@ -0,0 +1,4 @@
+# setup for LEDs on chan9
+SERVO9_FUNCTION 120
+
+GPS_DRV_OPTIONS 4
```

Comment:
this is not really required....we've done some testing and UBLOX GPS can be detected at the normal baud rate even on F405s...

Reply:
Removed